packages/packages/site-settings/src/components/portal.tsx

```diff
@@ -0,0 +1,24 @@
+import { PropsWithChildren } from 'react';
+import { Portal as BasePortal } from '@elementor/ui';
+import { isRouteActive, routeCloseEvent, routeOpenEvent, useListenTo } from '@elementor/v1-adapters';
+
+export default function Portal( { children }: PropsWithChildren<object> ) {
+	const containerRef = useListenTo( [
+		routeOpenEvent( 'panel/global' ),
+		routeCloseEvent( 'panel/global' ),
+	], () => {
+		return isRouteActive( 'panel/global' )
+			? { current: document.querySelector( '#elementor-panel-inner' ) }
+			: { current: null };
+	} );
```

Comment:
🔥 
Nice! I would've never thought about using it like this

Maybe extract to multiple variables for readability?

Reply:
Done